tests/erica_legacy/samples/grundsteuer_sample_data.py

```diff
@@ -284,25 +285,58 @@ def parse(self):
         return Vertreter.parse_obj(self.build())
 
 
+class SampleBruchteilsgemeinschaft(Builder):
+    name: str
+    adresse: SampleAdresse
+
+    def __init__(self):
+        super().__init__()
+        self.name = "Bruchteilsgemeinschaft Hogsmeade"
+        self.adresse = SampleAdresse().plz("08642").ort("Hogsmeade")
+
+    def with_name(self, name: str):
```

Comment:
So far the (loose) convention has been the following:

With the `with_` prefix when the field should be set with whatever non-None, nonempty value automatically, without the prefix for when the caller should provide the value. So this should be `name(self, name: str)`, whereas `with_adresse(sef)` below is already fine.

Reply:
Thanks for the explanation. The problem with that and the reason I didn't do it (besides not knowing the convention. :D) is, that if I name it "name" (hehe) it is shadowed by the attribute "name" set in __init__, which is string and cannot be called. So either I remove the default name (which I don't really want) or I give this method another name. I could also image "with_explicit_name" or something.